homeassistant/components/matrix/__init__.py

```diff
@@ -53,6 +55,9 @@
 CONF_WORD = "word"
 CONF_EXPRESSION = "expression"
 
+CONF_USERNAME_REGEX = "@[^:]*:.*"
```

Comment:
```suggestion
CONF_USERNAME_REGEX = "^@[^:]*:.*"
```

usernames must start with `@`, correct?

Reply:
Correct; I ended up copying the pre-existing regex from beforehand as-is. 😅 
Fixed!